chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -157,7 +157,7 @@ pub(crate) struct NetworkState {
     /// AccountsData for TIER1 accounts.
     pub accounts_data: Arc<accounts_data::Cache>,
     /// Connected peers (inbound and outbound) with their full peer information.
-    pub connected_peers: ConnectedPeers,
+    pub tier2: Pool,
```

Comment:
```suggestion
    pub tier2: connection::Pool,
```

or maybe rename to `ConnectionPool`.

I am split on prefix-less names. On the one hand, yeah, `connection::ConnectionPool` is stuttery. On the other hand:

* using `connection::Pool` requires more discipline 
* When using "go to symbol by fuzzy name" functionality, `CoPo` is much easier to type than `co::Po` (and the latter even isn't supported by LSP in the first place). 

My current rough feeling is to use `PrefixThing` by default, and `perfix::Thing` where `prefix` is a crate, and is explicitly designed to be imported qualified.

Reply:
This introduces the name/concept `tier2`, so that's a good place for strategic comment that gives a broader context.

Reply:
ack, I'd like this one (and many other modules) to be imported qualified.